### ISSUE TITLE
Hyperpixel (not 4) not working on RPi4 8GB

### ISSUE THREAD

**Issue by HughMungis** [pimoroni/hyperpixel#59]
fresh install of raspian and i get the following errors.
am i doing something wrong?
```
pi@raspberrypi:/opt/hyperpixel $ curl https://get.pimoroni.com/hyperpixel | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 35038  100 35038    0     0  79995      0 --:--:-- --:--:-- --:--:-- 79995

This script will install everything needed to use
HyperPixel

Always be careful when running scripts and commands copied
from the internet. Ensure they are from a trusted source.

This script should -- only be run on a Raspberry Pi with RPi OS --
other systems and SBCs are not supported and may explode!

If you want to see what this script does before running it,
you should run: 'curl https://get.pimoroni.com/hyperpixel'


This installer is for the OLD HyperPixel 3.5"
It is not compatible with HyperPixel 4 Rectangular or Weirdly Square.

Do you wish to continue? [y/N] y

Checking environment...
Updating apt indexes...
.....................

Checking hardware requirements...

Checking for dependencies...

Cloning Github repo locally...
Cloning into 'hyperpixel'...
remote: Enumerating objects: 29, done.
remote: Counting objects: 100% (29/29), done.
remote: Compressing objects: 100% (26/26), done.
remote: Total 29 (delta 0), reused 22 (delta 0), pack-reused 0
Receiving objects: 100% (29/29), 1.92 MiB | 869.00 KiB/s, done.

Finalising Install...

Installing dependencies...
python-rpi.gpio is required
Updating apt indexes...
.....................Installing python-rpi.gpio...
E: Package 'python-rpi.gpio' has no installation candidate
Apt failed to install python-rpi.gpio!
Falling back on pypi...
python-smbus is required
Installing python-smbus...
E: Package 'python-smbus' has no installation candidate
Apt failed to install python-smbus!
Falling back on pypi...

Installing Requirements...

Installing init script...

All done!

Some changes made to your system require
your computer to reboot to take effect.

Would you like to reboot now? [y/N] N

Some changes made to your system require
your computer to reboot to take effect.

Would you like to reboot now? [y/N] N
```

TL;DR
Apt failed to install python-rpi.gpio!
and 
Apt failed to install python-smbus!

are the only things throwing errors (that i know of)
